tests/integration_tests/tests/agentless_tests/test_runtime_functions.py

```diff
@@ -0,0 +1,143 @@
+########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+
+from integration_tests import AgentlessTestCase
+from integration_tests.tests.utils import get_resource as resource
+
+
+class TestRuntimeFunctionEvaluation(AgentlessTestCase):
+    BASE_VALUE = 'aaa'
+    CHANGED_VALUE = 'bbb'
+
+    def setUp(self):
+        super(TestRuntimeFunctionEvaluation, self).setUp()
+        self.bp_path = resource('dsl/deployment_update_functions.yaml')
+
+    def _assert_properties(self, deployment, value):
+        instances_node1 = self.client.node_instances.list(
+            node_id='node1', deployment_id=deployment.id)
+        self.assertEqual(len(instances_node1), 1)
+        instances_node2 = self.client.node_instances.list(
+            node_id='node2', deployment_id=deployment.id)
+        ni1 = instances_node1[0]
+        ni2 = instances_node2[0]
+
+        outputs = self.client.deployments.outputs.get(deployment.id)['outputs']
+        self.assertEqual(outputs['out1'], value)
```

Comment:
well it can make you think that it always will return value

Reply:
please reword this comment so that i can understand

Reply:
Sorry :) I meant that because you always return 'value' how can we be sure that it can return more than one value

Reply:
deployment outputs tests check that deployment outputs can return different values